bindings/python/examples/sponge.py

```diff
@@ -0,0 +1,40 @@
+from manifold3d import Manifold
+import numpy as np
+
+
+def fractal(holes, hole, w, position, depth, maxDepth):
+    w /= 3
+    holes.append(hole.scale([w, w, 1.0]).translate(
+        [position[0], position[1], 0.0]))
+    if depth == maxDepth:
+        return
+    offsets = np.array([
+        [-w, -w],
+        [-w, 0.0],
+        [-w, w],
+        [0.0, w],
+        [w, w],
+        [w, 0.0],
+        [w, -w],
+        [0.0, -w],
+    ])
+    for offset in offsets:
+        fractal(holes, hole, w, position + offset, depth + 1, maxDepth)
+
+
+def posColors(pos, _):
+    return [1 - p / 2 for p in pos] + [1.0]
+
+
+def run(n=1):
+    result = Manifold.cube([1, 1, 1], True)
+    holes = []
+    fractal(holes, result, 1.0, np.array([0.0, 0.0]), 1, n)
+
+    hole = Manifold.compose(holes)
+
+    result -= hole
+    result -= hole.rotate([90, 0, 0])
+    result -= hole.rotate([0, 90, 0])
+
+    return result.trim_by_plane([1, 1, 1], 0).set_properties(4, posColors).scale(100)
```

Comment:
How is the new number of properties 4? It seems like it would be 3, and anything else would trigger your invalid shape error below? Did you check that the resulting GLB looks like our logo?

Reply:
trimesh expects rgba for vert_color (and max is 255, but I did the conversion in the export part)

Reply:
Oh, my python is just weak - I totally missed that ` + [1.0]` was pushing an additional item into the array. Thanks!